src/main/scala/io/iohk/ethereum/jsonrpc/AkkaTaskOps.scala

```diff
@@ -0,0 +1,20 @@
+package io.iohk.ethereum.jsonrpc
+
+import akka.actor.{Actor, ActorRef}
+import akka.pattern.ask
+import akka.util.Timeout
+import monix.eval.Task
+
+import scala.reflect.ClassTag
+
+object AkkaTaskOps {
+  implicit class TaskActorOps(val to: ActorRef) extends AnyVal {
+
+    def askFor[A](
+        message: Any
+    )(implicit timeout: Timeout, classTag: ClassTag[A], sender: ActorRef = Actor.noSender): Task[A] =
+      Task
+        .fromFuture((to ? message).mapTo[A])
```

Comment:
Do we want to use strict evaluation here? Shouldn't be `Task.deferFuture`?

Reply:
Very good question 🤔

Reply:
This breaks the tests, I will explore what is needed to fix that.

Reply:
Changed to `Task.deferFuture`: https://github.com/input-output-hk/mantis/pull/756/commits/7494676cc21ff01b75df74207c35f6061c110bbe